### ISSUE TITLE
Unable to get code using git clone command

### ISSUE THREAD

**Issue by xddcore** [xddcore/SleepPanda#4]
Environment: windows system
Error description:

when run `git clone https://github.com/xddcore/SleepPanda.git`
The following error occurs:
```
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
```
This will make it impossible to obtain the source code of SleepPanda correctly.

**Comment by xddcore** [comment]
solution:
**Step1:**
Run the following code under git bash:
```
# ignore escape characters in the path
git config --global core.protectNTFS false
# Disable newline conversion
git config --global core.autocrlf false
# Chinese file name, garbled problem. If set to false, characters above 0x80 will not be quoted
git config --global core.quotepath false
#Allow field paths
git config --global core.longpaths true
```
**Step2:**
git bash run `git clone https://github.com/xddcore/SleepPanda.git`

**Step3:**
Enter the SleepPanda directory you just cloned.
git bash run `cd SleepPanda/`

**Step4:**
git bash run:
```
git reset
git checkout
```
**Step5:**
Use Github Desktop to manage code repositories.
Github Desktop -> File -> Add local repository -> **! ! ! Uncheck the files that show changes to avoid polluting the code repository**

**Comment by xddcore** [comment]
This git clone bug is fully fixed in commit `7f72d0ebd70230334c5c5df6fc043815f397c6ec`. (https://github.com/xddcore/SleepPanda/commit/7f72d0ebd70230334c5c5df6fc043815f397c6ec)
